libs/blocks/library-config/library-config.js

```diff
@@ -2,14 +2,14 @@ import { createTag } from '../../utils/utils.js';
 
 const LIBRARY_PATH = '/docs/library/library.json';
 
-async function loadBlocks(content, list, query) {
+async function loadBlocks(content, list, query, type) {
```

Comment:
With this many parameters I think is better to switch to passing an object. It helps avoid parameters related errors.
```suggestion
async function loadBlocks({ content, list, query, type }) {
```

Reply:
Done